src/dmd/typinf.d

```diff
@@ -53,7 +53,9 @@ extern (C++) void genTypeInfo(Type torig, Scope* sc)
         /* If this has a custom implementation in std/typeinfo, then
          * do not generate a COMDAT for it.
          */
-        if (!builtinTypeInfo(t))
+        if (t.ty == Terror)
+            t.vtinfo.errors = 1;
```

Comment:
Magic number "1"; what does that mean?  Is it a count of 1 or a integer masquerading as a boolean?  But, if this is the status quo in DMD, I suspect we can change that in another PR.

Reply:
`errors` has been converted to bool since, so the magic means "true". Updated.